### ISSUE TITLE
Indention problem in files containing tabs

### ISSUE THREAD

**Issue by sblask** [powerman/vim-plugin-AnsiEsc#10]
I have set my git to use vim as pager: `pager = vim -c 'set tabstop=4 nomodified nomodifiable nolist nowrap' -c 'AnsiEsc' -` so things like `git diff` is shown in `vim`. This works great in most cases, but has a hiccup when the diff contains tabs. Consider following diff:

```
[31m-	pager = vim -c 'set tabstop=4 nomodified nomodifiable nolist nowrap' -c 'AnsiEsc' -[m
[32m+[m	[32mpager = vim -c 'set tabstop=4 nomodified nomodifiable nolist nowrap' -[m
```

After calling `AnsiEsc` escape codes disappear and I see colors, but the second `pager` is not on the same level as the first. At first I thought there are suddenly two tabs or that `tabstop` is set in a bad way, but that's not the case. 

Is there anything that can be done about this?

**Comment by powerman** [comment]
> At first I thought there are suddenly two tabs or that tabstop is set in a bad way, but that's not the case.

Actually it is. Try `set ts=12` and they'll be at same level. This has nothing with AnsiEsc, it's how concealing Vim feature works - concealed symbols isn't shown, but they're there. This can be easily noticed while trying to move cursor left/right over concealed symbols - you'll need to press key several times to move cursor one position left/right. So, keeping in account concealed escape-codes counts as symbols we've much more symbols before TAB on second line than on first line. So, we need larger `tabstop` value to compensate this.

I didn't tried to check, vim has a lot of options, maybe there are one which can change this vim behaviour for concealed symbols. If you'll find one - please let me know. :)

**Comment by sblask** [comment]
Funky. Setting tabstop to something higher is not helpful as I don't know how many concealed characters there are, so it might still be off. I now changed my pager to `pager = sed 's/\t/    /g' | vim -c 'set nomodified nomodifiable nolist nowrap' -c 'AnsiEsc' -` so basically replacing tab by spaces - which is totally fine because I set `nolist` and they are not visible anyway. Cursor and movements are totally off anyway due to the concealed characters, but as I hadn't realized this until now, it doesn't seem to matter for me ;-) Thanks for the info!
